app/views/forms/odk_manifest.xml.erb

```diff
@@ -7,5 +7,15 @@
         <downloadUrl><%= url_for_path(@ifa.path) %></downloadUrl>
       </mediaFile>
     <% end %>
+
+    <% unless @questions.empty? %>
+      <% @questions.each do |q| %>
```

Comment:
Note you can decorate all of these at once with `DecoratorFactory.decorate_collection`

Reply:
whao!